util/regexp-assemble/data/920600.data

```diff
@@ -83,7 +86,10 @@ charse[^t{{non-token-chars}}]{{token-chars}}*
   ##!=> allowed-charsets
 |(?:
   ##!=>
-[^c{{non-token-chars}}]{{token-chars}}*
+##! Do not match space, otherwise this the following would be possible:
+##! "text/html; charset=invalid"
+##! `charset` would be matched by `{{token-chrs}}`
```

Comment:
```suggestion
##! `charset` would be matched by `{{token-chars}}`
```

Reply:
Oh... Well, that's another thing for the toolchain: check for unknown definitions.

Reply:
https://github.com/theseion/crs-toolchain/issues/39